src/elements/content-sidebar/__tests__/BoxAISidebar.test.tsx

```diff
@@ -24,18 +24,30 @@ describe('elements/content-sidebar/BoxAISidebar', () => {
         expect(screen.getByRole('heading', { level: 3, name: 'Box AI' })).toBeInTheDocument();
     });
 
-    test('should have accessible "Expand" button', () => {
+    test('should have accessible Agent selector if boxai.agentSelector.enabled is true', () => {
+        renderComponent({}, { 'boxai.agentSelector.enabled': true });
+
+        expect(screen.getByTestId('sidebar-agent-selector')).toBeInTheDocument();
```

Comment:
selecting by testid should be last resort, can we select by role and if not role by text?